### ISSUE TITLE
Pluto v0.19.34 broken on start

### ISSUE THREAD

**Issue by Reinis** [fonsp/Pluto.jl#2740]
After updating Pluto on Arch Linux and launching, it opens, but shows Loading.... on the home page for my and featured notebooks. When opening new notebook, it opens a blank page.

![attels](https://github.com/fonsp/Pluto.jl/assets/1250117/244943c2-9aea-49a2-91df-7a50bf9d7c8f)

There is an error in browser console. Reloading with clear cache doesn't help.
![attels](https://github.com/fonsp/Pluto.jl/assets/1250117/a951305c-f242-4d81-be47-cf495040184e)

**Comment by tlow22** [comment]
Same behavior on Mac as @marodriguezs mentioned. Currently working with 0.19.32 as a workaround.

**Comment by ablaom** [comment]
Same here.

```julia
julia> versioninfo()
Julia Version 1.9.1
Commit 147bdf428cd (2023-06-07 08:27 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin22.4.0)
  CPU: 12 × Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, skylake)
  Threads: 12 on 12 virtual cores
Environment:
  JULIA_LTS_PATH = /Applications/Julia-1.6.app/Contents/Resources/julia/bin/julia
  JULIA_PATH = /Applications/Julia-1.9.app/Contents/Resources/julia/bin/julia
  JULIA_EGLOT_PATH = /Applications/Julia-1.6.app/Contents/Resources/julia/bin/julia
  JULIA_NUM_THREADS = 12
  DYLD_LIBRARY_PATH = /usr/local/homebrew/Cellar/libomp/9.0.1/lib/
  JULIA_NIGHTLY_PATH = /Applications/Julia-1.9.app/Contents/Resources/julia/bin/julia
```

**Comment by axelparmentier** [comment]
Same here

Julia Version 1.9.4
Commit 8e5136fa297 (2023-11-14 08:46 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 16 × Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, skylake)
  Threads: 1 on 16 virtual cores
Environment:
  LD_PRELOAD = /usr/lib/x86_64-linux-gnu/libstdc++.so.6
  LD_LIBRARY_PATH = :/home/axel/apps/gurobi1001/linux64/lib

**Comment by Pangoraw** [comment]
Thank you for the report. It looks like the bundler failed for the 0.19.34 release, I will trigger a new release from https://github.com/fonsp/Pluto.jl/commit/343ea9780ad7877c976f1792632a3404beea0401 which doesn't have the same problem.

**Comment by Pangoraw** [comment]
This issue should be fixed by updating to Pluto@0.19.35.
